core/api/src/adapter.rs

```diff
@@ -100,6 +100,22 @@ where
         }
     }
 
+    async fn get_block_header_by_hash(
```

Comment:
It seems an useless interface.

Reply:
Yes. This was an legacy implementation that is no longer in use and has been deleted.